src/js/view/view.js

```diff
@@ -853,12 +852,20 @@ function View(_api, _model) {
     };
 
     function _getCurrentElement() {
-        return _model.get('isFloating') ? _wrapperElement : _playerElement;
+        return _model.get('isFloating') && _resizeOnFloat ? _wrapperElement : _playerElement;
```

Comment:
LGTM - can you just clarify why we need to check for `_resizeOnFloat` here as well? Note that `_getCurrentElement` is used to determine visibility etc. - just want to make sure this doesn't break.

Reply:
@vseventer Sure, I'm checking for ```_resizeOnFloat``` because i want to use the ```_playerElement``` in ```updateBounds``` instead of ```_wrapperElement``` when there is a player currently floating and the orientation of the device changes.

This is so floating player can be properly adjusted on either landscape or portrait based on the new dimensions of ```_playerElement```